rq_dashboard/templates/rq_dashboard/dashboard.js

```diff
@@ -226,212 +232,14 @@ var modalConfirm = function(action, cb) {
-        }
-
-        $.each(pagination.pages_in_window, function(i, page) {
-            var html = template_page(page);
-            var $el = $(html);
-
-            // Special markup for the active page
-            if (page.number === {{ page|tojson|safe }} ) {
-                $el.addClass('active');
-            }
-
-            $ul.append($el);
-        });
-
-        // next page
-        if (pagination.next_page !== undefined ) {
-            html = template_next_page(pagination.next_page);
-            $el = $(html);
-            $ul.append($el);
-        } else {
-            html = $('script[name=no-next-page-link]').html();
-            $ul.append(html);
-        }
-
-        // last page
-        html = template_last_page(pagination.last_page);
-        $el = $(html);
-        if (pagination.current_page == pagination.num_pages) {
-            $el.addClass('disabled');
-        }
-        $ul.append($el);
-
-        if (done !== undefined) {
-            done();
-        }
-    };
-
-    var refresh_table = function() {
-        if (window.getSelection().toString()) {
-            $('#alert-fixed').show();
-            return;
-        }
-        $('#alert-fixed').hide();
-        $('span.loading').fadeIn('fast');
-        reload_table(function() {
-            $('span.loading').fadeOut('fast');
-        });
-    };
-
-    $(document).ready(function() {
-
-        reload_table();
-        $('#refresh-button').click(refresh_table);
-        setInterval(refresh_table, POLL_INTERVAL);
-
-    });
-
-    // Enable the AJAX behaviour of the empty button
-    $('#empty-btn').click(function(e) {
-        e.preventDefault();
-        e.stopPropagation();
-
-        var $this = $(this);
-        modalConfirm('empty', function() {
-            $.post($this.attr('href'), function(data) {
-                reload_table();
-            });
-        });
-
-        return false;
-    });
-
-    $('#compact-btn').click(function(e) {
-        e.preventDefault();
-        e.stopPropagation();
-
-        var $this = $(this);
-        modalConfirm('compact', function() {
-           $.post($this.attr('href'), function(data) {});
-        });
-        return false;
-    });
-
-    $('#workers-btn').click(function(e) {
-        e.preventDefault();
-        e.stopPropagation();
-
-        $('#workers').toggle();
-
-        return false;
-    });
-
-    $('#requeue-all-btn').click(function(e) {
-        e.preventDefault();
-        e.stopPropagation();
-
-        var $this = $(this);
-        modalConfirm('requeue all', function() {
-            $.post($this.attr('href'), function(data) {});
-        });
-        return false;
-    });
-
-    // Enable the AJAX behaviour of the cancel button
-    $tbody.on('click', '[data-role=cancel-job-btn]', function(e) {
-        e.preventDefault();
-        e.stopPropagation();
+{{ job_table.render_js(queue.name, 'pending', page if (state == 'pending' or queue_name == 'failed') else 1)  }}
```

Comment:
Stopping my review here as I need to attend to other items. but will continue reviewing later. I want to give you my feedback up to this point.